pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: w_transport
-version: 3.2.8
+version: 3.2.9
```

Comment:
Could you back this change out? Our release bot commits the version bump

Reply:
Done